tests/test_hvac_csc.py

```diff
@@ -174,35 +136,38 @@ async def test_enable_on_all_subsystems_one_by_one(self):
                 remote=self.remote, state=salobj.State.ENABLED
             )
             for topic in HvacTopic:
-                if topic.value not in xml.TOPICS_ALWAYS_ENABLED:
+                if topic.value not in TOPICS_ALWAYS_ENABLED:
                     subsystem = topic.name
-                    # Retrieve the method to enable or disable a subsystem.
-                    enable_method = getattr(self.remote, "cmd_" + subsystem + "_enable")
+                    # Retrieve the DeviceId.
+                    device_id = DeviceId[subsystem].value
+                    data = {"device_id": device_id}
```

Comment:
You can skip the `.value` if you make it DeviceId an IntEnum (here and everywhere in your code)

Reply:
Done